system/src/ble_control_request_channel.h

```diff
@@ -123,6 +131,13 @@ class BleControlRequestChannel: public ControlRequestChannel {
     volatile bool subscribed_; // Set to `true` if the client is subscribed to the notifications
     volatile bool writable_; // Set to `true` if the TX characteristic is writable
 
+    bool initialized_;
+
+    hal_ble_uuid_t bleCtrlSvcUuid_{};
```

Comment:
Keep initializations uniform. Either in-class initialization or initialization in constructor initializer list.